devices/Axp192/Axp192.cs

```diff
@@ -502,6 +503,33 @@ public void EnableDCDC1(bool state)
             I2cWrite(Register.SwitchControleDcDC1_3LDO2_3, buf);
         }
 
+        /// <summary>
+        /// Gets or Sets the state of EXTEN switch control.
+        /// </summary>
+        /// <value>
+        /// <see langword="true"/> if EXTEN switch is enabled, <see langword="false"/> otherwise.
+        /// </value>
+        public bool EXTENEnable
```

Comment:
Not the preper naming :-( Should have been `ExternEnable` or even better `EnableExtern`